### PR TITLE
Migrate `/news` ->  `/blog` URLs in website

### PR DESCRIPTION
**Pull request by nuke-web3**
Fixes DR-202 - just ran a find&replace :magic_wand: 

Locally ran `lychee` against `/website` and have no errors, two unrelated fixes in the PR:
- there was a broken Notion link, removed
- theblock.com

### CODE REVIEW

File: website/lychee.toml [hunk 1]
```diff
@@ -8,4 +8,6 @@ max_retries = 3
 exclude = [
   # rate limited
   "https://epubs.siam.org/doi/10.1137/0108018",
+  # 403 for link checking only
+  "https://www.theblock.co",
```
Comment (nuke-web3) [inline website/lychee.toml:12]: ignored now, there are some sites hostile to lick checking (DDOS prevention) that 403

**Review by pdg744 — APPROVED** [review]

**Comment by flaub** [comment]
This seems better served as a server-side redirect/alias rather than updating all our content. I hope even without this change all the links still work.

**Comment by flaub** [comment]
The 1.1 docs got released, so this will need to be updated for this as well.

**Comment by nahoc** [comment]
> This seems better served as a server-side redirect/alias rather than updating all our content. I hope even without this change all the links still work.

I set up redirects on the website, so all `/news` URLs should already be redirecting to their `/blog` counterparts 👍

**Review by nahoc — APPROVED** [review]

File: website/docs/education.md [hunk 2]
```diff
@@ -23,6 +22,5 @@ Check out the [dev docs].
 [conference talks and podcast appearances]: https://www.youtube.com/playlist?list=PLcPzhUaCxlCgCvzkkaBWzVuHdBRsTNxj1
 [Dev Docs]: ../api/zkvm
 [Discord]: https://discord.gg/risczero
-[Education Database]: https://risczero.notion.site/5a335a1d29b44cb48c44b36ae66f366f?v=95cda72e39d3403fbfb59884afb0045d&pvs=4
```
[the remark this answers is not in the document]
Comment (pdg744) [reply website/docs/education.md:26]: @nahoc this link should be working now

**Comment by nuke-web3** [comment]
> This seems better served as a server-side redirect/alias rather than updating all our content. I hope even without this change all the links still work.

I always prefer updating to the correct link when it's easy to do. Curious why you would rather have a redirect over this PR? Happy to close if we want the redirects to stay in place.

File: website/docs/education.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (nuke-web3) [reply website/docs/education.md:26]: reverted 5973b80

**Pull request closed without merging**